app/views/gmp_main.scala.html

```diff
@@ -39,17 +39,14 @@
 
 @main(
   applicationConfig = config.ApplicationConfig,
-  title = title,
+  title = title + " - " + Messages("service.title") + " - " + Messages("gov.uk"),
```

Comment:
You could replace this with `s"$title - ${Messages("service.title")} - ${Messages("gov.uk")}"` it's up to you